src/hearings/services/hearings.service.ts

```diff
@@ -29,6 +30,18 @@ export class HearingsService {
       { caseReference: caseId });
   }
 
+  public cancelHearingRequest(hearingId: string, reasons: RefDataModel[]): Observable<any> {
+    const reasonIds: string[] = [];
+    reasons.forEach(reason => reasonIds.push(reason.key));
+    let httpParams = new HttpParams();
+    reasonIds.forEach(id => {
+      httpParams = httpParams.append('reasonkey', id);
+    });
+    return this.http.delete<any>(`api/hearings/cancelHearings?hearingId=${hearingId}`, {
+      params: httpParams
+    });
```

Comment:
is this following the body in the contract?

Reply:
Yeah I think so. Why?

Reply:
What do you want me to do @rayliangatos

Reply:
We don't have much time. @rayliangatos